cleancode/src/main/java/com/epam/cleancode/comments/InvalidInputException.java

```diff
@@ -1,16 +1,16 @@
 package com.epam.cleancode.comments;
 
-public class InvalidInputException extends RuntimeException{
+public class InvalidInputException extends RuntimeException {
+
     String message;
-    public InvalidInputException(String s) {
-        super();
-        this.message = s;
 
+    public InvalidInputException(String _message) {
```

Comment:
we don't use `_` prefix in java.
In this case we have 2 options:
1.  `this.message = message;`
2. `message = errorMessage`